partiql-conformance-tests/src/main.rs

```diff
@@ -0,0 +1 @@
+fn main() {}
```

Comment:
Is this needed because Cargo expects either a `bin` or `lib` and doesn't really expect an integration-test-only crate?